sentry-datasource-proxy/build.gradle.kts

```diff
@@ -0,0 +1,64 @@
+plugins {
+    `java-library`
+    kotlin("jvm")
+    jacoco
+    id(Config.QualityPlugins.errorProne)
+    id(Config.QualityPlugins.gradleVersions)
+}
+
+configure<JavaPluginConvention> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
```

Comment:
you can import `org.jetbrains.kotlin.gradle.tasks.KotlinCompile` and use `KotlinCompile`